extension/src/providers/gotoProvider.ts

```diff
@@ -0,0 +1,122 @@
+import * as vscode from 'vscode';
+import { AutoLispExt } from '../extension';
+import { Sexpression } from '../format/sexpression';
+import { ReadonlyDocument } from '../project/readOnlyDocument';
+
+interface LocatedItem {
+	line: number;
+	column: number;
+	fspath: string;
+}
+
+export class AutolispDefinitionProvider implements vscode.DefinitionProvider{
+	async provideDefinition(document: vscode.TextDocument, position: vscode.Position, token: vscode.CancellationToken): Promise<vscode.Location | vscode.Location[]> {
```

Comment:
This is very cool feature!!!
I test a little and it works cool for functions and variables!